src/saving/SaveHelper.cs

```diff
@@ -364,6 +384,8 @@ private static void PerformSave(InProgressSave inProgress, Save save)
         {
             save.SaveToFile();
             inProgress.ReportStatus(true, TranslationServer.Translate("SAVING_SUCCEEDED"));
+            SaveIsRecentlyPerformed = true;
+            recentSaveTimer.Start();
```

Comment:
```suggestion
            recentSaveTimer.Stop();
            recentSaveTimer.Start();
```
This is to reset the remaining duration if the timer is currently running

Reply:
I see, I was initially concerned about whether the timer automatically resets once Start() is called again after the first call (saving and then shortly saving again afterwards), but this clears it. Thanks!